source/translators/how-tos/how-to-request-a-new-language.rst

```diff
@@ -5,27 +5,36 @@ Assumptions
 ***********
 
 * The language you are interested in is not listed on the Transifex
-  `edx-platform project page`_.
+  `openedx-translations project page`_.
 
 * A variant of the language that may meet your needs is also not listed
   (``Chinese`` vs ``Chinese-China``).
 
+* Prefer a root locale over a variant locale (``de`` over ``de_DE``) whenever applicable so other locales can inherit
+  from it e.g. ``de_AT``.
 
 * If you request a new language, we ask that you commit to the success of your
   language’s translation project. Particularly, we expect you to:
 
-  * Be an active translator
+  * Be an active translator: Translate the ``edx-platform`` and main Micro-frontends such as Learning MFE and others.
```

Comment:
@ehuthmacher should we point to wherever you have documented requirements or are they covered here?

Reply:
@e0d @ehuthmacher This is a new requirement that I'm proposing. I suppose we need to further document it somewhere else.

Reply:
@ehuthmacher could you please check the changes proposed in this pull request?

Reply:
@OmarIthawi The policy looks good to me. I will run it by the group on Wednesday to ensure that they are satisfied as well. I added it to our TX GH board. The only correction is, under the requirements section - "be an active translator of the openedx-translations project", rather than edx-platform.

Reply:
@e0d All of our translation related documentation needs to be updated to reflect the TX project transition, under OEP-58. I will bring this up during our next TX meeting to see if someone has capacity to assist in this effort.